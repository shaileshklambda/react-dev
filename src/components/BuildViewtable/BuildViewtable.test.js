import React from "react";
import renderer from "react-test-renderer";
import Buildtable from "./index.js";

it("Buildtable: default", () => {
  const component = renderer.create(<Buildtable />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
