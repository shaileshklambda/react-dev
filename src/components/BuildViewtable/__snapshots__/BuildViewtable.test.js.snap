// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buildtable: default 1`] = `
<div>
  Invalid Data
</div>
`;
